lib/experimental/new_embed.dart

```diff
@@ -655,6 +666,10 @@ major browsers, such as Firefox, Edge (dev channel), or Chrome.
     textElement.remove();
   }
 
+  void _handleOpenInDartPad() {
+    window.open('https://dartpad.dev/$gistId', 'DartPad');
```

Comment:
Just tested this with /embed-dart.html?id=92dd64a18ebcaaac0e380a50cbd1b0c2 and was bitten by the fact that the playground added the code from `main.dart` but not `test.dart`, which left me with a non-working code state.

I'm not sure of the best way to handle this. Some options:

* Leave it as-is. Folks can just learn that the snippets in exercises with tests will need additional work in order to run.
* Leave as-is, but add a popup after the user clicks the Open in DartPad button warning them that the test code won't be going with them.
* Update the playground to concatenate the test code onto main.dart and present the result (possibly throwing in a comment above the test code to explain what's going on).

WDYT?

Reply:
That's a good point. I think the issue is that we don't support test code in the playground. Created https://github.com/dart-lang/dart-pad/issues/1348 to track.